networking-calico/networking_calico/plugins/calico/plugin.py

```diff
@@ -90,7 +90,7 @@ def __init__(self):
         #   File "/usr/lib/python3/dist-packages/oslo_db/api.py", line 142, in wrapper
         #     return f(*args, **kwargs)
         #   File "/usr/lib/python3/dist-packages/neutron_lib/db/api.py", line 183, in wrapped
-        #     LOG.debug("Retry wrapper got retriable exception: %s", e)
+        #     LOG.debug("Retry wrapper got retryable exception: %s", e)
```

Comment:
I missed commenting on the earlier occurrences, but I think we shouldn't change occurrences of "retriable" to "retryable".  For me that's not a clear enough "fix" to justify the change.  (Assuming you're happy with that, please revert all the similar "retriable" changes.)